src/README.md

```diff
@@ -1,24 +1,18 @@
-# Getting Set Up
-
-For this tutorial you will need a testnet account.  If you do not have one go [here](https://wallet.testnet.near.org), using the email option is the easiest method for ensuring you can recover your account.
+# Welcome to RAEN 🌧
 
-`raen` is a build tool for NEAR smart contracts.
+RAEN is a build tool for NEAR smart contracts.
 
 With RAEN, you can:
 
 * `build`: compile a contract, generate its Application Contract Interface (ACI), and inject it into a custom section of the contract's WebAssembly (Wasm) binary.
 * `fetch` _[coming soon]_: use the ACI of a deployed contract to generate source code bindings for cross contract calls and client interfaces.
 
+# Getting Set Up
 
-## Install `raen`
-
-To get started, first [install RAEN](https://raen.dev/docs/guide/installation.html).
-## Clone examples for repo (optional)
+[Install RAEN](https://raen.dev/docs/guide/installation.html), then install the examples repository to follow along:
 
 ```bash
-
 git clone --depth 1 --branch v0.0.3 https://github.com/raendev/examples.git --recursive
-
 ```
 
-If you have experience with NEAR and want to get to the RAEN-specific stuff you can skip to the [status message contract](./status-message/wit.md).
+If you have experience with NEAR, you can skip ahead to [Chapter 2: Status Message](./status-message/wit.md).
```

Comment:
I think perhaps they should skip all the way to the advanced.  Add a final page to the chapter with the full code and explanation.  This way they can see it in action without the hand holding.  Plus it can act as a summary for the new users.  So you could make a stub for now and link it here.

Reply:
Seems weird to link to a stub. Maybe we just remove this line until that chapter is done?

Reply:
Let's just comment it out and put a todo